ruby_library/entities/author.rb

```diff
@@ -0,0 +1,35 @@
+class Author
+    DEFAULT_NAME_RANGE = 3..15
+
+    attr_reader :name, :year, :bio
+
+    def initialize(name, year, bio='')
+        valid?(name, year)
+        @name = name
+        @year = year
+        @bio = bio
+    end
+
+    def to_s
+        "Author { #{@name}, #{@year}. #{@bio} }"
+    end
+
+    def ==(other)
+        if other.is_a?(Author)
+            @name == other.name && @year == other.year && @bio == other.bio
+        end
+    end
+
+    private
+
+    def valid?(name, year)
+        case
```

Comment:
please, use separate validations for each field and for each validations

Reply:
ok